GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h

```diff
@@ -93,7 +90,7 @@ typedef void (^GIDDisconnectCallback)(NSError *_Nullable error);
 ///
 /// @param callback The `GIDSignInCallback` block that is called on completion.  This block will be
 ///     called asynchronously on the main queue.
-- (void)restorePreviousSignInWithCallback:(nullable GIDSignInCallback)callback;
+- (void)restorePreviousSignInWithCallback:(nullable void (^)(GIDUserAuth *_Nullable userAuth, NSError *_Nullable error))callback;
```

Comment:
Since no auth flow is occurring here, I think we should return GIDGoogleUser rather than GIDUserAuth.

Reply:
Doen.